GitUI/Infrastructure/PuttyHelpers.cs

```diff
@@ -81,6 +78,12 @@ public static bool StartPageantIfConfigured(Func<string?> sshKeyFileLoader)
                 process.WaitForInputIdle();
             }
 
+            string? sshKeyFile = sshKeyFileLoader();
+            if (!string.IsNullOrWhiteSpace(sshKeyFile) && !File.Exists(sshKeyFile))
+            {
+                return false;
+            }
```

Comment:
❔ If a file is configured, we should tell the user if it cannot be found. 
```suggestion
            if (string.IsNullOrWhiteSpace(sshKeyFile))
            {
                return false;
            }

            ThrowIfFileNotFound(sshKeyFile, $"'{sshKeyFile}'", TranslatedStrings.ErrorSshKeyNotFound);
```

Reply:
Yes, thank you!